core/src/main/java/org/frankframework/management/bus/endpoints/ServerStatistics.java

```diff
@@ -101,6 +104,39 @@ public Message<String> getServerInformation(Message<?> message) {
 		return new JsonMessage(returnMap);
 	}
 
+	public static Long getFileSystemTotalSpace() {
+		try {
+			File systemDir = getSystemDir();
+			if (systemDir == null) return null;
+			return systemDir.getTotalSpace();
+		} catch ( Exception e ) {
+			log.debug("Caught Exception",e);
+			return null;
+		}
+	}
+
+	public static Long getFileSystemFreeSpace() {
+		try {
+			File systemDir = getSystemDir();
+			if (systemDir == null) return null;
+			return systemDir.getFreeSpace();
+		} catch ( Exception e ) {
+			log.debug("Caught Exception",e);
+			return null;
+		}
+	}
+
+	private static File getSystemDir() {
+		String dirName = System.getProperty("APPSERVER_ROOT_DIR");
```

Comment:
This used to be WebSphere logic, I think it's safe to remove. It might in fact be a (way) better option to use the log.dir because that's the partition that actually matters 😃

Reply:
only using log.dir from now